src/routes/blog/[slug]/+page.svelte

```diff
@@ -33,4 +44,19 @@
 		</hgroup>
 		<svelte:component this={data.content} />
 	</article>
+	<div class="box-border text-end text-sm">
```

Comment:
@koilebeit thank you very much for this PR. I'd order the platforms along their usage in our peer group which IMHO would be
- email
- whatsapp
- facebook
- linkedin
- x
Are there any numbers on telegram?

Reply:
Thanks for the review. I can change the order of the buttons.

But there is still the question of the exact placement and layout/design of the share buttons. Also the question of whether the buttons should only appear in the blog posts or also in the blog overview or elsewhere on the homepage?

Reply:
I'd move the bar up, over the blog content. But we should ask @mtwente for design questions